replica.yml

```diff
@@ -1,5 +1,12 @@
 ---
 version: "3.4"
+x-logging: &logging
+  logging:
+    driver: json-file
+    options:
+      max-size: 10m
+      max-file: "3"
```

Comment:
What the purpose of the logging definition? Control log size?

Reply:
Yes, that's to control log size and avoid filling local disk.